### ISSUE TITLE
App disables Direct Input

### ISSUE THREAD

**Issue by RonOren** [keymapperorg/KeyMapper#1225]
**Developer TODO**
- [ ] create new branch. put issue number at start of name if not a very quick fix.
- [ ] write tests. put issue number in comment
- [ ] update documentation
- [ ] merge and delete branch (don't squash because want commit history to see why I made changes)

**Accessibility disables Direct Input on OnePlus 12**
As it sounds,
When Accessibility is turned on Xbox controller connected to OnePlus Device via USB OTA cable doesn't work. When accessibility is disabled everything works as intended.
Tested on multiples apps, all behave the same.

**To Reproduce**
1. Use OnePlus 12 Phone
2. Enable Accessibility
3. Connect Xbox Controller
4. Open any kind of game with controller support

**Device Info:**
 - Device: OnePlus 12
 - Android version: 14

**Comment by sds100** [comment]
Can you try other apps that use an accessibility service, like Button Mapper?

**Comment by ronoren** [comment]
> Can you try other apps that use an accessibility service, like Button Mapper?

I have, I get the same issue.

**Comment by sds100** [comment]
Can you find _any_ accessibility service that doesnt have the issue? What about Talkback or Switch Access?

**Comment by ronoren** [comment]
> Can you find _any_ accessibility service that doesnt have the issue? What about Talkback or Switch Access? Otherwise it sounds like some bug or unintended behavior with the Direct Input feature that I probably can't fix.

Other accessibility services work just fine. The only other service which gives me the same result is "Button Mapper" by flar2.

**Comment by ronoren** [comment]
One thing that is important to note,
All the buttons work and are recognised when using [Gamepad Tester](https://hardwaretester.com/gamepad)

There are only 2 thing which doesn't work: analog triggers and sticks. (BTW, the stick click button does work)

The moment I disable the accessibility all analog inputs work.

**Comment by jambl3r** [comment]
Seems extremely niche and there doesn't seem to be a known solution either
